### PR TITLE
OboeTester: Update Test Output channel box and signal UI

### LINKED ISSUE #1587: OboeTester Test Output Channels and Signal UI show when they shouldn't
In Test Output, the bottom two things we can change seem to do the wrong thing if we change them at the wrong time.

For output channels, changing the channel checkbox between open and start seems to do nothing.

For signals, this does not do anything after a stream has started.

### PR DESCRIPTION
**Pull request by robertwu1**
Fixes #1587

### CODE REVIEW

File: apps/OboeTester/app/src/main/java/com/mobileer/oboetester/TestOutputActivity.java [hunk 1]
```diff
@@ -105,9 +103,17 @@ private void configureChannelBoxes(int channelCount) {
...
         configureChannelBoxes(0);
+        mNativeApiSpinner.setEnabled(true);
         super.closeAudio();
     }
 
+    public void startAudio() throws IOException {
+        super.startAudio();
+        int channelCount = mAudioOutTester.getCurrentAudioStream().getChannelCount();
+        configureChannelBoxes(channelCount);
+        mNativeApiSpinner.setEnabled(false);
```
Comment (philburk) [inline apps/OboeTester/app/src/main/java/com/mobileer/oboetester/TestOutputActivity.java:114]: mNativeApiSpinner is badly named, by me, so this fix is confusing.
Please rename to mOutputSignalSpinner or something better.
Also rename NativeApiSpinnerListener.

File: apps/OboeTester/app/src/main/java/com/mobileer/oboetester/TestOutputActivity.java [hunk 2]
```diff
@@ -105,9 +103,17 @@ private void configureChannelBoxes(int channelCount) {
 
     public void closeAudio() {
         configureChannelBoxes(0);
+        mNativeApiSpinner.setEnabled(true);
```
Comment (philburk) [inline apps/OboeTester/app/src/main/java/com/mobileer/oboetester/TestOutputActivity.java:106]: This stuff should be moved to stopAudio().

**Review by philburk — CHANGES_REQUESTED** [review]

File: apps/OboeTester/app/src/main/java/com/mobileer/oboetester/TestOutputActivity.java (hunk 1 above)
Comment (philburk) [reply apps/OboeTester/app/src/main/java/com/mobileer/oboetester/TestOutputActivity.java:114]: And in pauseAudio().

File: apps/OboeTester/app/src/main/java/com/mobileer/oboetester/TestOutputActivity.java (hunk 2 above)
Comment (philburk) [reply apps/OboeTester/app/src/main/java/com/mobileer/oboetester/TestOutputActivity.java:106]: And pauseAudio()

Comment (robertwu1) [reply apps/OboeTester/app/src/main/java/com/mobileer/oboetester/TestOutputActivity.java]: Changed to all 3. Seems like we need it for pause, stop, and close.

**Review by philburk — APPROVED** [review]

**Pull request merged**